docs/contributing.md

```diff
@@ -89,7 +89,7 @@ During development (adapt according to your preference):
 
     cargo watch --ignore '.jj/**' -s \
       'cargo clippy --workspace --all-targets \
-       && cargo +1.64 check --workspace --all-targets'
+       && cargo +1.70 check --workspace --all-targets'
```

Comment:
shouldn't having explicit `+1.70` be not necessary here as cargo would read the toolchain file?

Reply:
git review ui ate a large part of my comment (because I edited it), lol

I said that it's interesting that besides docs and CI (which can potentially read the toolchain file like Nix does) there are two sources of truth, the toolchain file and Cargo.toml, and cargo probably would read the latter as the toolchain file is more of a rustup thing iirc